### ISSUE TITLE
Manage Groups allows removal of masterlist rules

### ISSUE THREAD

**Issue by CptMcSplody** [Nexus-Mods/Vortex#5256]
**Describe the bug**
Assigning lines (Group afters) to masterlist groups allows for the removal of masterlist lines (group afters).

**To Reproduce**
1. Go to the plugins tab. 
2. Select _manage groups_.
3. Draw a line from A to B for two groups that are not connected.
4. Any masterlist lines to group B from earlier groups can now be removed.

**Expected behavior**
I expected it to only add a new line from A to B that one could later remove.

**Screenshots**

![image](https://user-images.githubusercontent.com/1944639/62710427-2e806700-b9ef-11e9-9c1b-1ed2a637b2f6.png)

![image](https://user-images.githubusercontent.com/1944639/62710622-7c956a80-b9ef-11e9-8fc8-d31a24528a29.png)

![image](https://user-images.githubusercontent.com/1944639/62710655-8d45e080-b9ef-11e9-81fe-5b96734a95b3.png)

![image](https://user-images.githubusercontent.com/1944639/62710690-a0f14700-b9ef-11e9-8370-d0a2d5d810fd.png)

![image](https://user-images.githubusercontent.com/1944639/62710708-ababdc00-b9ef-11e9-92d3-2f088f26febd.png)

**Platform (please complete the following information):**
 - OS: Windows 10 v1809, OS Build 17763.615
 - Vortex Version 1.0.0

**Additional context**

**Comment by TanninOne** [comment]
It doesn't actually remove the masterlist rule, the rule will still be used by LOOT, it's just not displayed in the group editor.

The reason this happens is because when I wrote the editor I worked under the assumption that once you have a group in the userlist it fully replaces the masterlist entry and thus Vortex would copy existing rules into the userlist and completely ignore masterlist rules.
The rules thereby all became editable because I thought that's what LOOT worked.

Whether that assumption was correct at the time I can't determine (or rather: it would cost time for no real benefit), it's definitively not correct now.
